packages/qcheck-alcotest/qcheck-alcotest.0.19/opam

```diff
@@ -28,4 +28,4 @@ url {
     "md5=6913306eb3065da1dee749d67732cb01"
     "sha512=ab77384e64887c4c88d6e641f433709a83bafc7cd2d05d548fdfee3593beb3bc5c4345db03e324e033ecd4fd2ae600fa92cea3e5753d5e322d13a01ba787eeb6"
   ]
-}
\ No newline at end of file
+}
```

Comment:
There's some end-of-line-at-end-of-file artifact here

```suggestion
}
```

Reply:
Yes, my vim automatically adds newlines at end of text files where they are missing. Pushed a commit to remove them, because for some reason Github can't apply your suggestions.